src/libutil/util.hh

```diff
@@ -137,6 +137,13 @@ N string2IntWithUnitPrefix(std::string_view s)
     throw UsageError("'%s' is not an integer", s);
 }
 
+/**
+ * Pretty-print a byte value, e.g. 12433615056 is rendered as `11.6
+ * GiB`. If `align` is set, the number will be right-justified
+ * (e.g. `__11.6 GiB`).
```

Comment:
The functional tests show no underscores. Let's not document the amount of padding.

```suggestion
 * GiB`. If `align` is set, the number will be right-justified by
 * padding with spaces on the left.
```